src/Application.php

```
@@ -54,38 +51,33 @@ class Application
         $this->environment = $this->container->get(Environment::class);
 
+        $componentRegistry = $this->container->get(LifecycleComponentRegistry::class);
+
         $events = $this->container->has(EventDispatcherInterface::class)
             ? $this->container->get(EventDispatcherInterface::class)
             : null;
 
-        $classDiscovery = $this->container->has(ClassDiscovery::class)
-            ? $this->container->get(ClassDiscovery::class)
-            : null;
+        $classDiscovery = $this->container->getOrNull(\Semitexa\Core\Discovery\ClassDiscovery::class);
```

PHPStan (2.1.46)

[error] Call to an undefined method Psr\Container\ContainerInterface::getOrNull().


(method.notFound)